pages/blog/hacktoberfest-2022.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Is Hacktoberfest Good For Maintainers?"
+date: 2020-11-05T06:00:00+01:00
+type: Community
+tags:
+  - Hacktoberfest
+cover: /img/posts/asyncapiXhacktoberfest.webp
+authors:
```

Comment:
please add `featured: true` so that this article comes at first place when posted on the website.